<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnnotationStatusFilterWidget</class>
 <widget class="QWidget" name="AnnotationStatusFilterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Annotation Status Filter</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayoutMain">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBoxStatusFilter">
     <property name="title">
      <string>アノテーション状態</string>
     </property>
     <layout class="QGridLayout" name="gridLayoutStatus">
      <property name="spacing">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBoxUnprocessed">
        <property name="text">
         <string>未処理</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelUnprocessedCount">
        <property name="text">
         <string>(0)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBoxProcessing">
        <property name="text">
         <string>処理中</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="labelProcessingCount">
        <property name="text">
         <string>(0)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkBoxCompleted">
        <property name="text">
         <string>完了</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="labelCompletedCount">
        <property name="text">
         <string>(0)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="checkBoxError">
        <property name="text">
         <string>エラー</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="labelErrorCount">
        <property name="text">
         <string>(0)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutActions">
     <item>
      <widget class="QPushButton" name="pushButtonSelectAll">
       <property name="text">
        <string>全選択</string>
       </property>
       <property name="toolTip">
        <string>全ての状態を選択</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSelectNone">
       <property name="text">
        <string>全解除</string>
       </property>
       <property name="toolTip">
        <string>全ての選択を解除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRefresh">
       <property name="text">
        <string>更新</string>
       </property>
       <property name="toolTip">
        <string>統計情報を更新</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>